### ISSUE TITLE
[Bug]: The selected prompt box is incorrect

### ISSUE THREAD

**Issue by HimiCos** [illixion/vscode-vibrancy-continued#76]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

<img width="240" alt="2023-05-18 18 42 33" src="https://github.com/illixion/vscode-vibrancy-continued/assets/92566148/7e30109d-ba4c-4a52-a51d-5e46917ba3ae">

### Expected Behavior

Normal display is OK

### Environment

```markdown
- OS: Mac
- VSCode: 1.78.2
- Extension: 1.1.15
- Theme: Github Dark
```

**Comment by illixion** [comment]
Please switch to a compatible theme like Dark+ and try again.

**Comment by HimiCos** [comment]
Yes, I tried to switch to Dark+, but the display is still abnormal

**Comment by HimiCos** [comment]
My native language is not English, so I may not express clearly. 

The problem is that there is indeed an extra character distance, and there are some minor flaws in the display such as the rounded corners. 

I uninstalled the plugin and took another screenshot for comparison. 
You can take a look.

<img width="225" alt="截屏2023-05-18 19 05 09" src="https://github.com/illixion/vscode-vibrancy-continued/assets/92566148/55e6614c-2783-4cd6-9f02-6d469cb447cf">

**Comment by illixion** [comment]
Thank you for clarifying, now I understand what you mean, I'll look into this.

**Comment by axaxxx** [comment]
hi i found a solution for this that works at least for me

in vscode_vibrancy.imports custom css add lines:
```

.lines-content.monaco-editor-background > .view-overlays > div > div.cslr.selected-text {
    background-color: rgba(0, 0, 0, 0) !important;
}

.lines-content.monaco-editor-background > .view-overlays.focused > div > div.cslr.selected-text {
    background-color: rgba(0, 0, 0, 0) !important;
}

.lines-content.monaco-editor-background > .view-overlays > div > div:last-of-type.cslr.selected-text,
.lines-content.monaco-editor-background > .view-overlays > div > div:nth-last-of-type(2).cslr.selected-text {
    background-color: rgba(252, 252, 250, 0.05) !important;
}

.lines-content.monaco-editor-background > .view-overlays.focused  > div > div:last-of-type.cslr.selected-text,
.lines-content.monaco-editor-background > .view-overlays.focused  > div > div:nth-last-of-type(2).cslr.selected-text {
    background-color: rgba(38, 79, 120,0.6) !important;
}
```

and in vscode settings add:
`"editor.guides.indentation": false`

**Comment by axaxxx** [comment]
> and in vscode settings add:
> `"editor.guides.indentation": false`

to preserve `"editor.guides.indentation": true` and only change it to `false` when text is being selected

i wrote a simple extension with `yo code` (selecting `New Extension (TypeScript)`)

tldr of this extension:
src/extension.ts:
```
import * as vscode from 'vscode';

export function activate(context: vscode.ExtensionContext) {

    console.log('vibrancy-fix-selection-preserve-editor-guides-indentation is active!');

    const config = vscode.workspace.getConfiguration('editor.guides');
    const isIndentationGuideEnabled = config.get('indentation');
    if (isIndentationGuideEnabled) {
        let editorselection = vscode.window.onDidChangeTextEditorSelection((e: vscode.TextEditorSelectionChangeEvent) => {
                if (e.selections[0] && !e.selections[0].isEmpty) {
                    config.update('indentation', false, true);
                } else {
                    config.update('indentation', true, true);
                }
        });
        context.subscriptions.push(editorselection);
    }
}

export function deactivate() {}
```

and add to package.json:
```
"activationEvents": [
    "onStartupFinished"
],
```

the ready .vsix file of extension in the attachment
[vibrancy-fix-selection-preserve-editor-guides-indentation-0.0.2.vsix.zip](https://github.com/illixion/vscode-vibrancy-continued/files/12822944/vibrancy-fix-selection-preserve-editor-guides-indentation-0.0.2.vsix.zip)

**Comment by AlexTemirbulatow** [comment]
I found out it only appears as long as there is some kind of transparency set to the "editor.background" color.
I set the color without any transparency, and it was normal again.
